package-lock.json

```diff
@@ -3371,7 +3371,7 @@
     },
     "readable-stream": {
       "version": "2.3.6",
-      "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
+      "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
```

Comment:
This change looks a bit strange. Otherwise looks good.

Reply:
Yeah I don't know why it was happening 😕 but it happened across multiple machines, so I just let it in.